cpu/nrf5x_common/periph/gpio.c

```diff
@@ -86,7 +86,7 @@ static inline NRF_GPIO_Type *port(gpio_t pin)
  */
 static inline int pin_num(gpio_t pin)
 {
-#ifdef CPU_MODEL_NRF52840XXAA
+#if defined(CPU_MODEL_NRF52840XXAA) || defined(CPU_MODEL_NRF52833XXAA)
```

Comment:
Now I wonder if this should be
```suggestion
#if defined(CPU_FAM_NRF52)
```
instead

Reply:
All adapted this way.